authentication/python/azure/README.md

```diff
@@ -0,0 +1,3 @@
+# To-do
+
+## Using the library
```

Comment:
provide the instructions.

Reply:
Done https://github.com/microsoft/kiota/pull/1108/commits/c29e1206529ded775a6b0702fc25293fd04a10dd